File: File/pascal.c
#include <stdio.h>

int main() {
	int st,r;
	printf("Enter the number of rows: \n");
	scanf("%d", &r);
	printf("\n");
	for(int i=2; i<=r+1; i++) {
		for(int k = 0; k<=r+1-i; k++) {
			printf(" ");
		}
		st=1;
		for(int j=0; j<i-1; j++) {
			if(j)
				st = (st*(i-j-1))/j;
			printf("%d ", st);
		}
	       printf("\n");	
	}
	return 0;
}

File: python2/pascal.py
#!/usr/bin/python3

t = int(input("Enter the number of rows: "))

for i in range(1,t+1):
    print(" "*(t-i), end="")
    j = 1
    for l in range(i):
        if l==0:
            print(j, end=" ")
        else:
            j = (j*(i-l))//l
            print(j, end=" ")
    print()
        
